### PR TITLE
Updating for riff 0.0.5 release

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: _docs/022-running-on-gke-with-rbac.md [hunk 1]
```diff
@@ -112,12 +112,11 @@ helm install projectriff/kafka \
 Watch kubectl for kafka to start running. You may need to wait a minute for the container images to be pulled, and for zookeeper to start first.
 
 ### install riff with RBAC
-Install the riff 0.0.4 helm chart on the `riff-system` namespace, with the release name `demo`.
+Install the riff 0.0.4 helm chart on the `riff-system` namespace, with the release name `control`.
```
Comment (scothis) [inline _docs/022-running-on-gke-with-rbac.md:115]: 0.0.4?

File: _docs/020-getting-started-on-gke.md [hunk 2]
```diff
@@ -104,21 +103,26 @@ watch -n 1 kubectl get po,deploy --namespace riff-system
...
+deploy/control-riff-function-controller   1         1         1            1           5m
+deploy/control-riff-http-gateway          1         1         1            1           5m
+deploy/control-riff-topic-controller      1         1         1            1           5m
+deploy/transport-kafka                    1         1         1            1           11m
+deploy/transport-zookeeper                1         1         1            1           11m
 ```
 
+### install the current riff CLI tool
+
+Starting with the 0.0.4 version we provide a CLI tool written in Go that makes it easy to create and deploy functions.
```
Comment (scothis) [inline _docs/020-getting-started-on-gke.md:123]: The release that first included the goCLI doesn't really matter, or that it's written in go

**Review by jldec — APPROVED** [review]
nice - thx. 🙇

File: _docs/022-running-on-gke-with-rbac.md (hunk 1 above)
Comment (jldec) [reply _docs/022-running-on-gke-with-rbac.md:115]: fixed in 13fbdd8

File: _docs/020-getting-started-on-gke.md (hunk 2 above)
Comment (jldec) [reply _docs/020-getting-started-on-gke.md:123]: fixed in 13fbdd8

**Pull request merged**